src/api/hooks/Group.tsx

```diff
@@ -0,0 +1,23 @@
+import { useMutation, UseMutationResult, useQuery, useQueryClient } from 'react-query';
+import API from 'api/api';
+import { Group, RequestResponse } from 'types/Types';
+
+const QUERY_KEY_GROUPS = 'groups';
+const QUERY_KEY = 'group';
```

Comment:
```suggestion
const GROUPS_QUERY_KEY = 'groups';
```
Du kan kombinere disse til 1 `GROUPS_QUERY_KEY`